BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -167,26 +162,17 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
 
                 switch updateState {
                 case .UpdateRequired:
-                    self.outofdateView.snp.remakeConstraints { (make) -> Void in
-                        make.left.equalTo(0)
-                        make.right.equalTo(0)
-                        make.top.equalTo(self.deadbeatView.snp.bottom)
-                        make.height.equalTo(42)
-                    }
+                    self.outofdateView.isHidden = false
                     self.outofdateLabel.isHidden = false
                     self.outofdateLabel.text = "This version of the Beeminder app is no longer supported.\n Please update to the newest version in the App Store."
                     self.collectionView?.isHidden = true
                 case .UpdateSuggested:
-                    self.outofdateView.snp.remakeConstraints { (make) -> Void in
-                        make.left.equalTo(0)
-                        make.right.equalTo(0)
-                        make.top.equalTo(self.deadbeatView.snp.bottom)
-                        make.height.equalTo(42)
-                    }
+                    self.outofdateView.isHidden = false
                     self.outofdateLabel.isHidden = false
```

Comment:
must both be hidden simultaneously?

Reply:
I think only the view. Removed the code to hide the label.